front/src/modules/activities/components/ActivityEditor.tsx

```diff
@@ -154,7 +154,12 @@ export const ActivityEditor = ({
   const debouncedUpdateTitle = debounce(updateTitle, 200);
 
   const updateTitleFromBody = (body: string) => {
-    const parsedTitle = JSON.parse(body)[0]?.content[0]?.text;
+    const bodyContent = JSON.parse(body);
+    const parsedTitle = Array.isArray(bodyContent)
+      ? bodyContent[0].type !== 'image'
+        ? bodyContent[0].content[0]?.text
+        : ''
+      : '';
```

Comment:
Temporarily handling the case if image was added as a first activity (block note does not have a `contents` property for upload image)